ath79-generic: add support for GL-iNet GL-AR300M-Lite

rotanid opened the pull request:
- [x] must be flashable from vendor firmware
  - [x] webinterface
  - [x] tftp
  - [ ] other: <specify>
- [x] must support upgrade mechanism
  - [x] must have working sysupgrade
    - [x] must keep/forget configuration (if applicable)
      *think `sysupgrade [-n]` or `firstboot`*
  - [x] must have working autoupdate
    *usually means: gluon profile name must match image name*
- [x] reset/wps/phone button must return device into config mode
- [x] primary mac should match address on device label (or packaging) (https://gluon.readthedocs.io/en/latest/dev/hardware.html#notes)
- wired network
  - [x] should support all network ports on the device
  - [x] must have correct port assignment (WAN/LAN)
- wifi (if applicable)
  - [x] association with AP must be possible on all radios
  - [x] association with 802.11s mesh must be working on all radios 
  - [x] ap/mesh mode must work in parallel on all radios
- led mapping
  - power/sys led (_critical, because led definitions are setup on firstboot only_)
    - [x] lit while the device is on
    - [x] should display config mode blink sequence 
(https://gluon.readthedocs.io/en/latest/features/configmode.html)
  - radio leds
    - [x] should map to their respective radio
    - [x] should show activity
  - switchport leds
    - [x] should map to their respective port (or switch, if only one led present) 
    - [x] should show link state and activity

rotanid commented:
@mweinelt you were right of course about docs, but adding a label leaves no notification :(

blocktrron commented on docs/user/supported_devices.rst:
```diff
@@ -192,7 +192,11 @@ ath79-generic
   - WiFi pro 1750x
 
-  * OCEDO
+* GL Innovations
```
Rename this to `GL-iNet`. I've never read the company calling themselves GL Innovations. If at all, this is long long ago.

Looks good otherwise.

rotanid replied:
i copied this from other target, renaming there too?

mweinelt replied:
Yep, lets do it.

rotanid replied:
done, please check

blocktrron replied:
@rotanid It's nitpicky, but shouldn't it be `GL.iNet` instead of `GL-iNet`? (As I've proposed in my first commit)

rotanid replied:
i don't mind how it's called, you suggested to use GL-iNet, now it's GL.iNet, and mweinelt is also in the loop.
please come to an agreement about it and i will do it.

mweinelt replied:
-> `GL.iNet`

blocktrron replied:
@rotanid Sorry, I've mistyped my proposal in the initial commit. Realized it just now.

LGTM

rotanid replied:
done, please check and maybe merge

mweinelt commented on docs/user/supported_devices.rst:
```diff
@@ -37,13 +37,13 @@ ar71xx-generic
-  - GL-iNet 6408A (v1)
-  - GL-iNet 6416A (v1)
+  - GL.iNet 6408A (v1)
+  - GL.iNet 6416A (v1)
```
```suggestion
  - 6416A (v1)
```

mweinelt commented on docs/user/supported_devices.rst:
```diff
@@ -37,13 +37,13 @@ ar71xx-generic
   - GL-AR750
-  - GL-iNet 6408A (v1)
-  - GL-iNet 6416A (v1)
+  - GL.iNet 6408A (v1)
```
```suggestion
  - 6408A (v1)
```

mweinelt replied:
https://openwrt.org/toh/hwdata/gl.inet/gl.inet_6408a

rotanid commented:
@mweinelt implemented your suggestion, changed alphabetical order accordingly and removed the version number of 64xx as there's no other version out there.
